package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "casper-blockexplorer",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
You might've ran `npm install` in the root.  That would make sense for this file to be added here.  However we don't need a package.json in the root currently.  Can you delete this file?